### ISSUE TITLE
Provide a compare function to avoid sorting elements alphabetically.

### ISSUE THREAD

**Issue by manusa** [manusa/electronim#335]
## Description

The following code has been marked as a [bug](https://sonarcloud.io/project/issues?open=AYh_YKwWW2wJeQETzpxf&id=manusa_electronim):

https://github.com/manusa/electronim/blob/fa1d2767dd33d5f9757d938aa8a0b6383c8767cf/src/spell-check/dictionary.renderer/dictionary.worker.js#L36

It should be changed to:

```javascript
  return Array.from(ret.values()).sort((w1, w2) => w1.localeCompare(w2)).slice(0, 10);
```

**Comment by saffi-786** [comment]
Hey! So do you need help just to change that particular line of code?

**Comment by manusa** [comment]
Yes, nice and easy

**Comment by Rhythm-08** [comment]
If @saffi-786 is not working on this issue , So can I attempt this @manusa ?

**Comment by saffi-786** [comment]
@manusa I want to work on this issue. Can you plz assign it to me?

**Comment by manusa** [comment]
> @manusa I want to work on this issue. Can you plz assign it to me?
> 

You've got it.

**Comment by saffi-786** [comment]
I have added a PR #337 
Please review it

**Comment by Srajan-Bansal** [comment]
Hey! I have fixed #335
Can you review it.

**Comment by Y0ooSibbi** [comment]
Hey [manusa](https://github.com/manusa) , Can you please assign this to me if available?

**Comment by manusa** [comment]
Fixed by #337
